### DISCUSSION TITLE
Improve redirection in former handsontable/formula.js repo

### DISCUSSION THREAD

**Discussion by nicolashefti** [handsontable/hyperformula#809] [General]
### Description

I am Nicolas, maintainer of [@formulajs/formulajs](https://www.npmjs.com/package/@formulajs/formulajs).

Your former package is still heavily used [@handsontable/formulajs](https://www.npmjs.com/package/@handsontable/formulajs). In addition to the current statement about HypeFormula, I think this would be great to inform the developers that there is a continuity of development for Formula.js.

### Links

Documentation page to improve: https://github.com/handsontable/formula.js
An example: https://github.com/sutoiku/formula.js

### Type of issue

Other issues, suggestions, or ideas

**Comment by krzysztofspilka** [comment]
Hi @nicolashefti, thanks for bringing this up.

First off, good job on maintaining https://github.com/formulajs/formulajs. We know how hard the formulas can be:)

As for your improvement suggestion for the notification in our fork of the Sutoiku project - if you don't mind we'd rather keep it our way. Originally, we created the fork solely for `hot-formula-parser` needs, which was a part of Handsontable's plugin. Thus, we always considered it an internal project, not a general-purpose solution.

Also, we've put a lot of time and effort to create HyperFormula, and we'd like to encourage developers to give it a try.

I believe that the disclaimer at https://github.com/sutoiku/formula.js is sufficient for most users to recognize your work at https://github.com/formulajs/formulajs as a continuity of the project.

**Reply by nicolashefti** [reply]
Hi Chris,

Thanks for the quick answer. I understand your point about inviting developers using Formulajs to try Hyperfomula. Your new library brings everything to another level. Great job you did there!

However, Hyperformula does not serve the same kind of purpose as Formulajs. It's far more advanced. Swapping from one to another is not so obvious to me.

Plus, from an open-source project perspective, I remember in the last months the maintainers of handsontable/formula.js pointing at formulajs/formulajs in the new issues to re-direct developers. I just wanted this link to be more clear now the repo is archived.

The decision is yours in the end. I don't want to still fame nor community, but simply help developers using Formulajs to get updates to their package.

Nicolas
